build_tools/travis/install.sh

```diff
@@ -52,7 +52,7 @@ if [[ "$DISTRIB" == "conda" ]]; then
             ${PANDAS_VERSION+pandas=$PANDAS_VERSION}
     fi
     source activate testenv
-    # FIXME: required to test spectral clustering
+    # required to test spectral clustering
```

Comment:
Please do it the same way as pandas, i.e. if PYAMG_VERSION does not exist then do not install it, else do install it.

While you are at it, bonus points if you can improve the duplication in the conda create environment, e.g. something along these lines:

```bash
TO_INSTALL="python=$PYTHON_VERSION pip pytest pytest-cov numpy=$NUMPY_VERSION ..."

if [[ "$INSTALL_MKL" == "true" ]]; then
    TO_INSTALL="$TOINSTALL mkl"
fi

if [[ -n $PANDAS_VERSION ]]; then
    TO_INSTALL=""$TO_INSTALL pandas=$PANDAS_VERSION"
fi

cond create -n testenv --yes $TO_INSTALL

source activate testenv
if [[ -n $PYAMG_VERSION ]]; then
    pip install pymag=$PYAMG_VERSION
fi
```

I'd rather install from PyPI than from conda-forge. I have seen weird things happening when mixing conda and conda-forge (e.g. updating your numpy from conda-forge and all packages dependent on numpy).